config/cmd/cmd.go

```diff
@@ -221,13 +230,19 @@ var (
 		"quic":   quic.NewTransport,
 	}
 
+	DefaultRuntimes = map[string]func(...runtime.Option) runtime.Runtime{
+		"local": runtime.NewLocalRuntime,
+		"k8s":   runtime.NewK8sRuntime,
```

Comment:
- local => runtime.NewRuntime
- rename "k8s" to "kubernetes"
- have kubernetes runtime in its own runtime/kubernetes package

Reply:
`kubernetes` is effing long. Remember, these are cli flags. But, ok. Will change, boss.

Reply:
Can't put `kubernetes` in its own package because that will create import cycle:
Runtime requires `Option` to pass in `runtime.Notifier`, and `runtime.Service`.
Need to think about how to do this, but it's not an easy file move -- I would have done that already if it were.

Reply:
Not sure why it would create an import cycle. Kubernetes package importing the go-micro/runtime should cause zero issues

Reply:
All our other things follow this pattern with no issues

Reply:
Ah yeah, I forgot the `NewRuntime` function in `runtime` package no longer does the switch on type of runtime -- that's moved to `config/cmd` now, so should be ok, actually.